cybershell/validation_framework.py

```
@@ -0,0 +1,973 @@
+    def detect(self, 
+              evidence: List[ValidationEvidence],
+              exploitation_result: Dict) -> List[str]:
+        """Detect false positive indicators"""
+        
+        indicators = []
+        
+        # Check evidence for false positive patterns
+        for e in evidence:
+            # Check for generic errors
+            if e.evidence_type == EvidenceType.ERROR_MESSAGE:
+                if self._is_generic_error(e.data):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)